UI/css/ledgersmb.css

```diff
@@ -73,11 +73,15 @@ table.warning {
     border: 0;
 }
 
-#company_info, #version_info, #login_info {
+#version_info {
    font-weight: bold;
    font-size: 9pt;
 }
 
+#company_info, #login_info {
```

Comment:
Your call. They need to be in one place. I just fixed the layout but kept the actual distribution. This should be addressed in another PR.